src/App.tsx

```diff
@@ -250,6 +250,21 @@ function Toggle(props: {
       >
         {t("Bugs")}
       </button>
+      <button
```

Comment:
Note to self, I should probably clean this up so there isn't so much copy paste. This component should just take an array options. I'll do this later!